content/en/real_user_monitoring/rum_explorer.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Views Explorer
+kind: documentation
+description: ""
+further_reading:
+- link: "/real_user_monitoring/rum_analytics"
+  tag: "Documentation"
+  text: "Build analytics upon your events."
+---
+
+The Views Explorer allow you to explore all your views collected from your different applications.
+
+## Context
+
+Build up a context to explore your views in your Views explorer page first by selecting the proper [time range](#time-range) then by using the [search bar](#search-syntax) to filter your views and Analytics.
+
+### Time Range
+
+It appears directly under the search bar as a timeline. The time range feature allows you to display view in the view steam or Analytics within a given time period.
```

Comment:
```suggestion
It appears directly under the search bar as a timeline. The time range feature allows you to display view in the view steam or analytics within a given time period.
```

Reply:
Not sure if **view steam** is meant to be steam or stream, so left as is. Flagging just in case.